server/endpoints_v2.go

```diff
@@ -117,6 +127,12 @@ func (server *HTTPServer) addV2EndpointsToRouter(router *mux.Router) {
 	).Methods(http.MethodGet)
 }
 
+// addV2RedisEndpointsToRouter method registers handlers for endpoints that depend on our Redis storage
+// to provide responses.
+func (server *HTTPServer) addV2RedisEndpointsToRouter(router *mux.Router, apiPrefix string) {
+	router.HandleFunc(apiPrefix+StatusOfRequestID, server.readStatusOfRequestID).Methods(http.MethodGet)
```

Comment:
I called the method "getRequestStatusForCluster" in the openAPI. I don't know if we are using any docs generator or something similar, in which case it could be confusing. just FYI

Reply:
@epapbak if you mean the `operationId` in the OpenAPI spec, it doesn't correspond to the names of our functions. I just checked and out of 42 (nice) `operationId`, only 1 actually matches the handler function name (GET /rule). AFAIK it's only used in some way by IQE tests, but I might be wrong

btw you're missing an operationId definition [here ](https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/server/api/v2/openapi.json#L400), I can add it in the next PR if you want

Reply:
updated for the peace of mind :)

Reply:
FYI I'm adding the missing `operationId` in the next PR